SPECS/ocaml-findlib/ocaml-findlib.spec

```diff
@@ -49,54 +71,44 @@ cat src/findlib/ocaml_args.ml
   -sitelib `ocamlc -where` \
   -mandir %{_mandir} \
   -with-toolbox
-make all
+%make_build all
 %ifarch %{ocaml_native_compiler}
-make opt
+%make_build opt
 %endif
 rm doc/guide-html/TIMESTAMP
-
-
+	
 %install
 # Grrr destdir grrrr
-mkdir -p %{buildroot}%{_bindir}
-mkdir -p %{buildroot}%{_mandir}/man{1,5}
+mkdir -p $RPM_BUILD_ROOT%{_bindir}
```

Comment:
I think there's no need to revert back to `$RPM_BUILD_ROOT`.

Reply:
Fixed